spring-resteasy/README-source.adoc

```diff
@@ -64,6 +64,13 @@ WARN  [org.jboss.as.ee] (MSC service thread 1-5) WFLYEE0007: Not installing opti
 WARN  [org.jboss.as.ee] (MSC service thread 1-5) WFLYEE0007: Not installing optional component org.springframework.web.context.request.async.StandardServletAsyncWebRequest due to an exception (enable DEBUG log level to see the cause)
 ----
 
+This warning can be ignored too:
```

Comment:
@emmartins Do I need to modify the `README.adoc` too? It seems like it's the expansion to the `README-source.adoc`

Reply:
No, only the README-source.adoc, that one is auto built.